workflows/raredisease.nf

```diff
@@ -153,6 +155,16 @@ workflow RAREDISEASE {
     )
     ch_versions = ch_versions.mix(CALL_SNV_DEEPVARIANT.out.versions)
 
+    CALL_SV_MANTA (
```

Comment:
```suggestion
    // TODO: Move this to a SV calling workflow 
    CALL_SV_MANTA (
```

Reply:
Yup! I'll update this branch to accommodate: https://github.com/Clinical-Genomics/raredisease/tree/feature/call_structural_variants